otherlibs/stdune/src/compact_position.mli

```diff
@@ -0,0 +1,47 @@
+(** Positions information that can be encoded within a single immediate *)
```

Comment:
Why is this not just `position.mli`? The "compact" bit is just an implementation detail, right?